packages/SymbioticController/src/Adapter/Nette/Template/TemplateRenderer.php

```diff
@@ -35,7 +35,9 @@ public function renderFileWithParameters(string $file, array $parameters = []):
         $latte = $template->getLatte();
 
         $layout = $this->guessLayoutFromFile($file);
-        $this->presenterHelper->setLayout($layout);
+        if ($layout) {
+            $this->presenterHelper->setLayout($layout);
```

Comment:
This might cause problems when rendering more than one file - if the second file has no layout it will use the layout of the first file which not expected.